src/content/en/updates/2020/06/devtools.md

```diff
@@ -0,0 +1,200 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Style editing for CSS-in-JS frameworks, Lighthouse 6.0, new JavaScript features, and more.
+
+{# wf_updated_on: 2020-06-24 #}
+{# wf_published_on: 2020-06-24 #}
+{# wf_tags: chrome85, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Style editing for CSS-in-JS frameworks, Lighthouse 6.0, new JavaScript features, and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 85) {: .page-title }
+
+{% include "web/_shared/contributors/jecelynyeen.html" %}
+
+## Style editing for CSS-in-JS frameworks {: #css-in-js }
+
+[cssom]: https://drafts.csswg.org/cssom/
+[constructable]: /web/updates/2019/02/constructable-stylesheets
+[shadowdom]: /web/fundamentals/web-components/shadowdom
+
+The Styles pane now has better support for editing styles that were created with the
+[CSS Object Model][cssom] (CSSOM) APIs. Many CSS-in-JS frameworks and libraries
+use the CSSOM APIs under the hood to construct styles.
+
+Styles added in JavaScript using [Constructable Stylesheets](constructable) are
+also now editable. Constructable Stylesheets are a new way
+to create and distribute reusable styles when using [Shadow DOM][shadowdom].
+
+{# https://chromium.googlesource.com/chromium/src.git/+/4609670fb4303585928d9784840459f04ca91f03 #}
+
+Chromium issue [#946975](https://crbug.com/946975)
+
+## Lighthouse 6 in the Lighthouse panel {: #lighthouse }
+
+[changelog]: https://github.com/GoogleChrome/lighthouse/releases/tag/v6.0.0
+[lighthouse]: https://web.dev/lighthouse-whats-new-6.0/
+
+The Lighthouse panel is now running Lighthouse 6.
+Check out [What's New in Lighthouse 6.0](lighthouse) for a summary of all the major
```

Comment:
I put the What's New post up here because I would rather direct attention to that than the full release notes